menus.xrc

```diff
@@ -382,11 +382,19 @@
     </object>
     <object class="wxMenuItem" name="add_cell">
         <label>_Add cell\tCtrl-+</label>
+        <extra-accels>
+            <accel>Shift-Ctrl-+</accel>
+            <accel>Ctrl-Num-+</accel>
```

Comment:
I've finally tested this (better late than never...) and this just doesn't seem to work, we need to use `Ctrl-Num +` for this accelerator to be triggered by the "+" key on the numpad. Am I missing something here and this was somehow broken by 3396a40015 (Mark keyboard key names translatable in a specific context, 2022-04-24) in wx in the meanwhile or did it really never work at all?